city_scrapers/spiders/chi_ssa_48.py

```diff
@@ -0,0 +1,109 @@
+from datetime import datetime
+
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa48Spider(CityScrapersSpider):
+    name = "chi_ssa_48"
+    agency = "Chicago Special Service Area #48 Old Town"
+    timezone = "America/Chicago"
+    start_urls = ["https://oldtownchicago.org/ssa-48/"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        meeting_year = response.xpath("//div[@class='meeting-dates-block']/h2")
+        meeting_date = response.xpath("//div[@class='meeting-dates-block']/h5")
+        meeting_info = response.xpath("//div[@class='meeting-dates-block']/p")
+        meeting_links = response.xpath("//div[@class='meeting-minutes-block']")
+
+        for item_date, item_info in zip(meeting_date, meeting_info):
+
+            start_time, end_time, parsed_meeting_date = self._parse_start_end(
+                item_date, item_info, meeting_year
+            )
+
+            meeting = Meeting(
+                title="Commission",
+                description="",
+                classification=COMMISSION,
+                start=start_time,
+                end=end_time,
+                all_day=False,
+                time_notes="",
+                location=self._parse_location(item_info),
+                links=self._parse_links(parsed_meeting_date, meeting_links),
+                source=self._parse_source(response),
+            )
+
+            meeting["status"] = self._get_status(
+                meeting, text=meeting_info.xpath(".//text()").get()
+            )
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_start_end(self, date, info, year):
+        meeting_year = year.xpath("//div[@class='meeting-dates-block']/h2/text()").get()
+        parse_year = meeting_year.split(" ")
+        year = parse_year[0]
+
+        meeting_date = date.xpath(".//text()").get()
+        meeting_time = info.xpath(".//text()").get()
+        parse_time = meeting_time.split("-")
+
+        start_time = datetime.strptime(
+            year + " " + meeting_date + " " + parse_time[0] + " " + parse_time[1][-2:],
+            "%Y %A %B %d %I:%M %p"
+        )
+
+        end_time = datetime.strptime(
+            year + " " + meeting_date + " " + parse_time[1], "%Y %A %B %d %I:%M%p"
+        )
+
+        parsed_meeting_date = datetime.strptime(year + " " + meeting_date, "%Y %A %B %d")
+
+        return start_time, end_time, parsed_meeting_date
```

Comment:
`parsed_meeting_date` isn't necessary here because `start_time` and `end_time` are `datetime` objects with the date embedded already. You can pass `start.date()` to the `_parse_links` function and then compare it to `meeting_minutes_date.date()` (since `strptime` always returns a `datetime`)

Reply:
Thank you for the hint.